keyboards/aleblazer/zodiark/config.h

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021 Spencer Deven <splitlogicdesign@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xF901
+#define PRODUCT_ID      0xF902
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Aleblazer
+#define PRODUCT         Zodiark
+#define MASTER_LEFT
+
+#ifdef OLED_DRIVER_ENABLE
+  #define OLED_TIMEOUT 400000
+#endif
```

Comment:
Is this is defined at the keyboard level, it probably doesn't need to be fined here.
```suggestion
```